### PR TITLE
Update Set-CsOnlineLisSubnet.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@CarolynRowe 
Adding information for how to define Lis Subnet

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsOnlineLisSubnet.md [hunk 1]
```diff
@@ -27,14 +27,16 @@ Enhanced 9-1-1 allows an emergency operator to identify the location of a caller
 
 The location ID which is associating with the subnet is not required to be the existing location.
 
+LIS subnets must be defined by the Network ID matching the subnet IP range assigned to clients. For example, the network ID for a client IP/mask of 10.10.10.150/25 is 10.10.10.128. For more information, see [Understand TCP/IP addressing and subnetting basics](https://docs.microsoft.com/en-us/troubleshoot/windows-client/networking/tcpip-addressing-and-subnetting)
```
Comment (get-itips) [inline skype/skype-ps/skype/Set-CsOnlineLisSubnet.md:30]:
```suggestion
LIS subnets must be defined by the Network ID matching the subnet IP range assigned to clients. For example, the network ID for a client IP/mask of 10.10.10.150/25 is 10.10.10.128. For more information, see [Understand TCP/IP addressing and subnetting basics](/troubleshoot/windows-client/networking/tcpip-addressing-and-subnetting)
```

**Review by dariomws — APPROVED** [review]
I'm ok once the URL is simplified. Thanks!

**Review by tseward — APPROVED** [review]
Approved once the URL is shortened.

**Comment by yogkumgit** [comment]
@carolynblanding Please review and update. Thanks

**Comment by yogkumgit** [comment]
@carolynblanding gentle reminder

**Comment by carolynblanding** [comment]
What is request to me here?

Regards,
Carolyn Blanding

**Comment by garycentric** [comment]
## [Before]

image

## [After]

image

**Pull request merged**